### PR TITLE
feat: Add clean command to dev.sh script

### PR DESCRIPTION
**Pull request by iNem0o**
Hello

This PR add a new development helper command to completely clean up the Docker environment. This is useful for contributors who want to reset their development environment to a clean state.

The `clean` command performs the following:
- Stops and removes all containers
- Removes all Docker images used by the project
- Removes all associated volumes
- Cleans up any orphaned containers

Usage:
```sh
./dev.sh clean
```

Documentation has been adjusted accordingly to the change.

### CODE REVIEW

File: dev.sh [hunk 1]
```diff
@@ -341,6 +347,6 @@ action=${1-}
 if [[ -n $@ ]];then shift;fi
 
 case $action in
-    build|up|down|ps|checks|test_all|unittest|test|composer_update|notice) do_$action "$@";;
+    build|up|down|ps|checks|test_all|unittest|test|composer_update|notice|clean) do_$action "$@";;
```
Comment (SimonMellerin) [inline dev.sh:350]: May be you can add a little description above (like line 334 just after `unittest ...`

**Review by SimonMellerin — CHANGES_REQUESTED** [review]
Thanks a lot for this PR, it's nice to see people use this dev tool and improve it!

**Review by SimonMellerin — APPROVED** [review]
@pounard  may be you want to review this one

**Comment by pounard** [comment]
It seems all good to me!

Pipeline is green, so let's merge this.

**Comment by pounard** [comment]
@iNem0o thanks!

**Comment by pounard** [comment]
It lack a line in the changelog, we'll have to add it.

**Pull request merged**